avro/src/types.rs

```diff
@@ -478,6 +480,15 @@ impl Value {
                     None
                 }
             }
+            (&Value::Fixed(n, _), &Schema::Uuid(UuidSchema::Fixed(_))) => {
```

Comment:
```suggestion
            (&Value::Fixed(n, _), &Schema::Uuid(UuidSchema::Fixed(size, ..))) => {
```
Should we check that `size` is also `16` here ?